frontend/apps/reader/modules/readergesture.lua

```diff
@@ -243,6 +249,8 @@ function ReaderGesture:init()
     local mirrored_if_rtl = {
         tap_top_left_corner = "tap_top_right_corner",
         tap_right_bottom_corner = "tap_left_bottom_corner",
+        double_tap_left_side = "double_tap_right_side",
+        double_tap_right_side = "double_tap_left_side",
```

Comment:
(I don't remember why I just set tap_top_left_corner  and tap_right_bottom_corner, and not their counterparts.
Oh, but I switch them, so one of the swapped pair is enough. But it's strange I made that kinda crossed, and not both: left_corner = "right" ...)